src/views/v2/rsvp/ari/form/fields/email.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Block: RSVP ARi
+ * Form Email
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/fields/email.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+/**
+ * Set the default value for the email on the RSVP form.
+ *
+ * @param string
+ * @param Tribe__Events_Gutenberg__Template $this
+ *
+ * @since 4.9
+ */
+$email = apply_filters( 'tribe_tickets_rsvp_form_email', '', $this );
+?>
+<div class="tribe-common-b1 tribe-tickets__form-field tribe-tickets__form-field--required">
+	<label
+		class="tribe-common-b2--min-medium tribe-tickets__form-field-label"
```

Comment:
you should probably place the `tribe-common-b2--min-medium` class on the same element as the one with `tribe-common-b1`